decidim-initiatives/app/views/decidim/initiatives/admin/committee_requests/index.html.erb

```diff
@@ -12,7 +12,7 @@
     </div>
 
     <div class="row column">
-      <div class="committee_link" id="committee_link">
+      <div id="committee_link">
```

Comment:
This should be a data attribute: 
```suggestion
      <div data-committee_link>
```

And we also need to change the JS selector:
```javascript
./decidim-initiatives/app/packs/src/decidim/initiatives/admin/invite_users.js            
8:      let link = document.querySelector("div[data-committee_link]"),
```

(I didn't actually tried it but I guess this work 😄)

Reply:
Addressed in [1d801b2](https://github.com/decidim/decidim/pull/12140/commits/1d801b23bfa059a884008a4228950428ad94bfd1)